docs/website/mtouch-errors.md

```diff
@@ -841,6 +841,12 @@ This generally indicates that there is a problem with your Xamarin.iOS installat
 
 <h3><a name="MT2015"/>MT2015: Invalid HttpMessageHandler `*` for watchOS. The only valid value is NSUrlSessionHandler.</h3>
 
+This is a warning that occurs because the project file specifies an invalid HttpMessageHandler.
+
+Earlier versions of Xamarin Studio generated by default an invalid value in the project file.
```

Comment:
That's likely true for XVS templates too. Maybe _Earlier versions of our preview tools ..._ ?